### PR TITLE
Move platform dependency to Swift Package Manager

### PR DESCRIPTION
**Pull request by kozlowsqi**
This moves the platform dependency from code to swift package manager,
meaning the @available clause can be dropped in sources.

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -1,30 +1,27 @@
...
 
 import PackageDescription
 
 let package = Package(
     name: "SegmentedPicker",
+    platforms: [
+        .iOS(.v13),
+        .macOS(.v10_15),
+        .watchOS(.v6),
+        .tvOS(.v13)
```
Comment (kieranb662) [inline Package.swift]: Drag Gesture not available on tvOS. Other than that, looks good

**Review by kieranb662 — CHANGES_REQUESTED** [review]
`DragGesture` not available on TVOS. Other than that great job, Thanks!

File: Package.swift (hunk 1 above)
Comment (kozlowsqi) [inline Package.swift]:
```suggestion
```

Comment (kozlowsqi) [reply Package.swift]: I suggest we remove it then from the supported platforms for now.. 
 Note that the tvOS picker looks like: 

image

And shouldn't be too hard to support

**Comment by kozlowsqi** [comment]
this still active?

**Pull request closed without merging**